### DISCUSSION TITLE
How can I hide this working directory line from terminal?

### DISCUSSION THREAD

**Discussion by tapasdatta** [zed-industries/zed#21814] [Feature Requests]
How can I hide this whole line from terminal?

<img width="658" alt="Screenshot 2024-12-10 at 10 09 30 PM" src="https://github.com/user-attachments/assets/0411410f-cf26-4e74-b4b6-344e154df795">

**Comment by 5brian** [comment]
You can add this to your config:
```
{
  "terminal": {
    "toolbar": {
      "breadcrumbs": false
    }
  }
}
```
See: https://zed.dev/docs/configuring-zed#terminal-toolbar

**Reply by tapasdatta** [reply]
Thank you!  It works.
